serverless.yml

```diff
@@ -0,0 +1,29 @@
+useDotenv: true
+
+service: serverless-slack-bot
+frameworkVersion: '3'
+
+
+package:
+  include:
+    - build/**
+    - node_modules/**
+
+provider:
+  name: aws
+  runtime: nodejs20.x
+  region: eu-north-1
+  environment:
+    SLACK_SIGNING_SECRET: ${env:SLACK_SIGNING_SECRET}
+    SLACK_BOT_TOKEN: ${env:SLACK_BOT_TOKEN}
+
+functions:
+  slackEventHandler:
+    handler: build/recieveSlackEvent.handleSlackEvent
```

Comment:
do not point handlers to the build files, point it to the ts file. rename file to slackEventHandler and put it under src/functions folder. put slackbot file under src/functions/utils